src/services/source/nativeIntegration.ts

```diff
@@ -25,21 +24,23 @@ export default class NativeIntegrationSourceService implements IntegrationSource
     return metaTO;
   }
 
+
```

Comment:
We should not expect any changes in this file actually

Reply:
done